spyder/utils/tests/data/example.py

```diff
@@ -0,0 +1,63 @@
+"""
+    ********************* VerySimpleWebBrowser ************************
+
+    This is a Very Simple Web Browser implemented over Qt and QtWebKit.
+
+    author: Juan Manuel Garcia <jmg.utn@gmail.com> ~/home
+
+    *******************************************************************
+"""
```

Comment:
Exactly @ccordoba12 , the tests that I designed and the logic are already able to filter this type of Headers that can be habitual in users of C and C ++